assets/src/domain/eventEditor/services/apollo/mutations/datetimes/updateTicketCache.ts

```diff
@@ -43,14 +44,15 @@ const updateTicketCache = ({ proxy, datetimeIn, datetimeId, action }: CacheUpdat
 		data,
 		variables: {
 			where: {
-				datetimeIn: newDatetimeIn,
+				datetimeIn: sortBy(identity, newDatetimeIn),
```

Comment:
I'm assuming this simply sorts the `newDatetimeIn` list of datetime IDs ?

Reply:
Yes